workinggroups/README.md

```diff
@@ -2,19 +2,30 @@
 
 ## Introduction
 
-The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a TOC member and voted with a super majority vote by the CNCF TOC. The TOC can also shut down a working group with a super majority vote.
+CNCF Working Groups (WGs) serve several purposes, for example to study and report on a particular question, to make recommendations based on their findings, or to complete a specific work package. The intention of WGs is _not_ to host a full project or specification. The work result of a WG may include a new project proposal, landscape, whitepaper report detailing their findings, or similar.
 
-## Process
+WGs must be created under the umbrella of a [CNCF Technical Advisory Group](https://github.com/cncf/toc/tree/main/tags) (TAG), and sponsored by a CNCF TOC member. In case of disagreement within TOC, a supermajority vote may be called 
```

Comment:
Can a WG be created under the umbrella of multiple TAGs?

Reply:
@nikhita I don't see why not, i also don't believe we should prevent a TAG's WG from being joint with another foundation.  However there must be a very clearly defined governance of which entity is the "home" for the WG to resolve issues as well as define CoCC and other governance, license, and process escalation related items.

Reply:
I think in the past we have addressed the overlaps but identifying "Related Work Areas" and "Other groups that the WG interacts with".  There will always be overlap, I believe it should be fine as long as the community has clarity as to what that is.